exercises/concept/locomotive-engineer/locomotive_engineer_test.rb

```diff
@@ -15,7 +15,8 @@ def test_get_array_of_wagons_with_one_inputs
   end
 
   def test_get_array_of_wagons_with_many_inputs
-    assert_equal [1, 10, 6, 3, 9, 8, 4, 14, 24, 7], LocomotiveEngineer.generate_list_of_wagons(1, 10, 6, 3, 9, 8, 4, 14, 24, 7)
+    assert_equal [1, 10, 6, 3, 9, 8, 4, 14, 24, 7],
+      LocomotiveEngineer.generate_list_of_wagons(1, 10, 6, 3, 9, 8, 4, 14, 24, 7)
```

Comment:
Cool; I'll squash everything on my branch, ensure the `[no important files changed]` tag is in the specific commit and remove the whitespace.

I have a terrible habit of fixing unrelated things "while I'm there" that my ADHD does not help me avoid 😳

Reply:
Ok, that change has been removed; I have squashed everything into a single commit and force-pushed the corrected commit. (Twice because I forgot to give it a summary line the first time 🤦🏼 )

I'm wandering off for a little now, but feel free to give me further feedback if it's still not quite right.